### ISSUE TITLE
Understanding Conformance 2.1 only talks about 2.0

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#967]
https://www.w3.org/WAI/WCAG21/Understanding/conformance

Understanding Conformance 2.1 only talks about 2.0

WCAG 2.0 is mentioned 11 times, WCAG 2.1 zero

**Comment by alastc** [comment]
The conformance model/content hasn't changed since 2.0. It is still applicable, but I take the point it could be confusing. 

We could add a note to the effect it hasn't change, or do a search & replace. I'll check with Michael there isn't an impact I don't know about.

**Comment by alastc** [comment]
Comment from Michael: Better to update the references to 2.1, and put a note that it is the same as 2.0.

**Comment by jake-abma** [comment]
@alastc wrote:

> Comment from Michael: Better to update the references to 2.1, and put a note that it is the same as 2.0.

Still see 2.0 all over, needs to be changed?

**Comment by patrickhlauke** [comment]
would it perhaps not be easier to just generalise the wording and say "WCAG"? it should be fairly implict to the reader that since they're reading documentation about WCAG 2.0 or WCAG 2.1 at that point, when it says "WCAG" it refers to "the spec that you're currently reading". make it future proof/less update burden/no need for weird "applies to 2.1 and 2.0 and ..."

**Comment by jake-abma** [comment]
Yeh, agree until WCAG 3.x is released and this is outdated..  :-) 

Maybe WCAG 2 series?

**Comment by patrickhlauke** [comment]
maybe i'm giving readers too much credit in understanding that what they read in one stable version doesn't apply to the next version. but yes, "2.x" sounds best to me

**Comment by patrickhlauke** [comment]
@alastc do we still want to tackle this sort of thing for WCAG 2.1, or just leave it and concentrate purely on 2.2 stuff? (I believe we fixed this in 2.2 not so long ago)
